pkg/hbone/README.md

```diff
@@ -0,0 +1,71 @@
+# HTTP Based Overlay Network (HBONE)
+
+HTTP Based Overlay Network (HBONE) is the protocol used by Istio for communication between workloads in the mesh.
+At a high level, the protocol consists of tunneling TCP connections over HTTP/2 CONNECT, over mTLS.
```

Comment:
I would skip 'over mTLS', and add 'over HTTP/2 and H3 CONNECT' ( not clear if QUIC is mTLS, or if in a trusted networks we must have outer mTLS or H2C is allowed )

Reply:
For now, isn't it mTLS? I'd rather keep the text representative of what has been built rather than what might be built in the future.